### PR TITLE
Subrepository: setup instructions

### PR DESCRIPTION
**Pull request by glaubinix**

### CODE REVIEW

File: docs/setup-subrepository.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+$client->subrepositories()->addOrEditTeams($subrepositoryName, [$team]);
+```
+
+### Granting external collaborators access to a subrepository
+
+Working on a Composer project with external developers might sometimes require that those developers get access to a subrepository
+to make sure they can run composer install and update and help manage the project's dependencies. Adding them as subrepository
+collaborators will only grant them access to selected subrepositories and not give them access to you your Private Packagist organization.
+
+You can invite via email subrepository collaborators on the subrepository's team page via the "Manage Collaborators" button.
```
Comment (naderman) [inline docs/setup-subrepository.md:150]:
```suggestion
Working on a Composer project with external developers might sometimes require that they get access to a subrepository
to make sure they can run composer install and update and help manage the project's dependencies. Adding them as subrepository
collaborators will only grant them access to selected subrepositories and will not give them access to you your Private Packagist organization.

You can invite subrepository collaborators via email on the subrepository's team page via the "Manage Collaborators" button.
```

Comment (naderman) [reply docs/setup-subrepository.md:150]: No snippet for API use here?

**Review by naderman — CHANGES_REQUESTED** [review]

File: docs/setup-subrepository.md (hunk 1 above)
Comment (glaubinix) [reply docs/setup-subrepository.md:150]: There is no API for this

File: docs/setup-subrepository.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# Private Packagist Subrepositories
+##
```
[the remark this answers is not in the document]
Comment (naderman) [reply docs/setup-subrepository.md:2]: Ahh!

File: docs/setup-subrepository.md (hunk 1 above)
Comment (naderman) [reply docs/setup-subrepository.md:150]: Guess we generally don't do user management through API so far, so can leave this until it becomes necessary.

**Comment by glaubinix** [comment]
Thanks @naderman ! This looks great, ready to merge and link it in the app?

**Review by naderman — APPROVED** [review]

**Comment by naderman** [comment]
@glaubinix yup :-)

**Pull request merged**